docs/user-docs/latest/plugins/installing-plugins-via-marketplace.md

```diff
@@ -4,37 +4,41 @@ description: Instructions to install new plugins in a Strapi application via the
 
-The Marketplace is a section of the admin panel that lists all plugins that can be installed in a Strapi application. The Marketplace is accessible from ![Marketplace icon](../assets/icons/marketplace.svg) _Marketplace_ in the main navigation of the admin panel.
+ ::: strapi
+The Marketplace in the admin panel only displays v4 plugins, but all plugins for all Strapi versions are discoverable in the [Strapi Market](market.strapi.io). Keep in mind that v3 and v4 plugins are not cross-compatible.
+:::
 
-<!--
+![The Marketplace interface](../assets/plugins/marketplace-v4.png)
 
-::: note
-Plugins can also be installed via the Command Line Interface (see [Developer Documentation](/developer-docs/latest/developer-resources/cli/CLI.md#strapi-install)).
-:::
+Plugins are displayed on individual cards containing:
 
-![Interface of the Marketplace](../assets/plugins/marketplace.png)
+- the plugin name, sometimes followed by either of the following badges:
+   - ![made by Strapi icon](../assets/icons/official-market.svg) to indicate the plugin is made by Strapi,
+   - ![verified by Strapi icon](../assets/icons/verified-marketplace.svg) to indicate the plugin was verified by Strapi.
+- the plugin description
+- a **Learn more** button for additional information, including detailed implementation instructions
+- a **Copy install command** button to copy the plugin installation command to the local clipboard
 
-The Marketplace displays each available plugin in a box, which contains:
-- the name of the plugin,
-- the description of the plugin,
-- indications on the current status of the plugin:
-   - "Compatible with your app": indicates that the plugin is not installed yet but can be installed on your Strapi application
-   - "Already installed": indicates that the plugin is already installed and available in your Strapi application
+In the top right corner of the Marketplace, the **Submit your plugin** button redirects to the Strapi Market where it is possible to submit your own plugin.
 
 ::: tip
-Click on the link icon ![External link icon](../assets/icons/external_link.svg) next to the name of a plugin to be redirected to the plugin package in the Strapi GitHub repository.
+The search bar displays find as you type results that search the plugin name and description.
 :::
 
+## Installing Marketplace plugins
+
 To install a new plugin via the Marketplace:
 
 1. Go to the ![Marketplace icon](../assets/icons/marketplace.svg) Marketplace.
-2. Among the available plugin, choose the one you wish to install.
-3. Click on the **Install** button in the chosen plugin's box.
+2. Choose an available plugin and click on the **Copy install command** button.
+3. Switch to your teminal and navigate to the Strapi application directory.
+4. Paste and execute the copied install command.
```

Comment:
```suggestion
4. Paste and run the copied install command.
```

I would suggest "run", this is what we use elsewhere (e.g. in dev docs) and what I've read most often over the web. 
Plus, though this is a bit of nitpicking, technically we as users don't execute the command, the terminal does 😆